test/integration/proposal-schemeregistrar.ts

```diff
@@ -0,0 +1,78 @@
+import * as uuid from "uuid";
+import { getContractAddresses } from "./utils";
+
+describe("SchemeRegistrar Proposals", () => {
+    let daoAddress: string;
+    let addresses;
+
+    before(() => {
+      addresses = getContractAddresses();
+      daoAddress = addresses.Avatar.toLowerCase();
+    });
+
+    it("Create a proposal, vote for it, stake on it", async () => {
+      const url = `/dao/${daoAddress}/`;
+      await browser.url(url);
+
+      browser.debug();
```

Comment:
remember to remove

Reply:
yeah, the test is not rly written yet anyway :-)